src/app/features/public-form/public-form-config.ts

```diff
@@ -1,14 +1,54 @@
 import { Entity } from "../../core/entity/model/entity";
 import { DatabaseEntity } from "../../core/entity/database-entity.decorator";
 import { DatabaseField } from "../../core/entity/database-field.decorator";
+import { LongTextDatatype } from "app/core/basic-datatypes/string/long-text.datatype";
 import { FieldGroup } from "app/core/entity-details/form/field-group";
 
+/**
+ * Each entity of this type defines a new publicly accessible form
+ * that can be reached through the given route even by users without being logged in.
+ */
 @DatabaseEntity("PublicFormConfig")
 export class PublicFormConfig extends Entity {
-  @DatabaseField() title: string;
-  @DatabaseField() description: string;
-  @DatabaseField() entity: string;
-  @DatabaseField() columns: FieldGroup[];
+  static override label = $localize`:PublicFormConfig:Public Form`;
+  static override labelPlural = $localize`:PublicFormConfig:Public Forms`;
+  static override route = "admin/public-form";
```

Comment:
```suggestion
  static override route = "admin/public-form";
  static override toStringAttributes = ["title"];
```

adding human-readable "name" to entity